test/pacmacs-test.el

```diff
@@ -126,3 +126,34 @@
 
      (should (null pacmacs--ghosts))
      (should (= 11 (length pacmacs--terrified-ghosts))))))
+
+(ert-deftest pacmacs--align-score-record-nickname-test ()
+  (let ((pacmacs--max-score-nick-size 5)
```

Comment:
Yep, just wanted to show that the function also depends on a global constant. And yes, elisp bindings are always dynamic. And in my opinion that was the main reason why this dialect of lisp sucked. But recently they introduced lexical-binding mode, which makes let bindings dynamic only for global variables. Which is bit better.